src/Magnum/Platform/Test/CMakeLists.txt

```diff
@@ -47,7 +47,9 @@ if(WITH_EMSCRIPTENAPPLICATION)
         # Test that the canvas and keylistener can be found with both the old
         # and the new Emscripten target behaviors
         # TODO: use target_link_options() once we can require CMake 3.13
-        "-s DISABLE_DEPRECATED_FIND_EVENT_TARGET_BEHAVIOR=0")
+        "-s DISABLE_DEPRECATED_FIND_EVENT_TARGET_BEHAVIOR=0"
+        # Enable memory runtime checks
+        "-s ASSERTIONS=2 -s SAFE_HEAP=1")
```

Comment:
Are there any downsides of enabling assertions? I suppose it will increase the resulting JS file size quite a bit -- I occasionally use these tests to check changes in build sizes, so I guess I'll have to remember to temporarily comment this out when doing the comparisons.

I was thinking about enabling this only for Debug builds, but I personally almost never build for Emscripten in Debug, so that wouldn't help in catching new breakages.

Reply:
You could enable it for the multiple canvas test app only, so it's tested but you still have the normal application to test with?

Reply:
Eh, let's keep it there :) If it gets in the way of something, I'll remove it, but for now better safe than sorry.